src/api/app/views/webui/comment/_new.html.haml

```diff
@@ -4,3 +4,10 @@
     = hidden_field_tag :commentable_id, commentable.id
     ~ f.text_area :body, size: '80x4', onkeyup: 'sz(this);', onclick: 'sz(this);', placeholder: 'Add a new comment (markdown markup supported)', required: 'required'
     = f.submit 'Add comment'
+- else
+  %h3
+    Login required, please
+    = link_to "login", session_new_url
+    or
+    = link_to "signup", user_register_user_url
+    in order to create comments
```

Comment:
The 'create' is a bit weird. Simply having `in order to comment` sounds more natural.